docs/scripts.md

```diff
@@ -1,6 +1,12 @@
 # Node.js scripts
 
-With Execa, you can write scripts with Node.js instead of a shell language. It is [secure](#escaping), [performant](#performance), [simple](#simplicity) and [cross-platform](#shell).
+With Execa, you can write scripts with Node.js instead of a shell language. [Compared to Bash and zx](#differences-with-bash-and-zx), this is more:
+  - [performant](#performance)
+  - [cross-platform](#shell): [no shell](../readme.md#shell-syntax) is used, only JavaScript.
+  - [secure](#escaping): no shell injection.
+  - [simple](#simplicity): minimalistic API, no [globals](#global-variables), no [binary](#main-binary), no [builtin CLI utilities](#builtin-utilities).
+  - [featureful](#simplicity): all Execa features are available ([process piping](#piping-stdout-to-another-command), [IPC](#ipc), [transforms](#transforms), [background processes](#background-processes), [cancellation](#cancellation), [local binaries](#local-binaries), [cleanup on exit](../readme.md#cleanup), [interleaved output](#interleaved-output), [forceful termination](../readme.md#forcekillafterdelay), etc.).
+  - [convenient to debug](#debugging): [verbose mode](#verbose-mode), [detailed errors](#errors), [error messages and stack traces](#cancellation), stateless API.
```

Comment:
Let's highlight **all the things** Execa can do that alternatives cannot. There are lots of them!